packages/filebrowser/style/base.css

```diff
@@ -127,17 +132,20 @@
 |----------------------------------------------------------------------------*/
 
 .jp-DirListing {
-  flex: 1 1 auto;
-  display: flex;
-  flex-direction: column;
   outline: 0;
 }
 
 .jp-DirListing-header {
-  flex: 0 0 auto;
-  display: flex;
-  flex-direction: row;
-  align-items: center;
+  display: grid;
+
+  /* TODO: Deduplicate with content grid */
+  grid-template-columns:
```

Comment:
While `grid-template-columns` may be *the* solution for now, it would be much nicer if we could use `grid-auto-columns` to allow for column reordering (together with setting size explicitly on elements). This or we need to generate `grid-template-columns` in JS.